src/outfile/CMakeLists.txt

```diff
@@ -41,6 +41,16 @@ file(COPY ${CMAKE_CURRENT_BINARY_DIR}/swmm_output_export.h
 )
 
 
+set_target_properties(swmm-output
+    PROPERTIES
+        MACOSX_RPATH TRUE
```

Comment:
Does the MACOSX_RPATH cover both macos and linux?

Reply:
No. It only covers Mac OS X. Rpath on Linux and Mac are slightly different.